src/main/java/org/isf/permissions/manager/GroupPermissionManager.java

```diff
@@ -33,7 +33,7 @@
 @Component
 public class GroupPermissionManager {
 
-	private GroupPermissionIoOperations operations;
+	private final GroupPermissionIoOperations operations;
```

Comment:
I do not see the value of adding `final`.  The variable is already `private` and adding `final` doesn't add anything to how the variable can be used.   In addition, if it is added, then there are hundreds of other occurrences like this in the code base that should be changed to match this pattern.   It isn't worth it IMHO.

Reply:
Variables which values are set via DI (Dependency Injection) should be final as there are initialized once.

Reply:
Thank you @SilverD3, could you please open a "task" for this so we don't forget?

Reply:
Sure